meta-lmp-base/recipes-security/optee/optee-os-fio.inc

```diff
@@ -21,7 +21,7 @@ CVE_PRODUCT = "linaro:op-tee op-tee:op-tee_os"
 PROVIDES = "virtual/optee-os"
 
 SRCBRANCH ?= "master"
-SRC_URI = "git://github.com/foundriesio/optee_os.git;protocol=https;branch=${SRCBRANCH}"
+SRC_URI = "git://github.com/ricardosalveti/optee_os.git;protocol=https;branch=${SRCBRANCH}"
```

Comment:
did you want this coming from your repo?

Reply:
I will update once the final patch set is tested with this PR (still got a couple more patches to add).